x/contractmanager/types/expected_keepers.go

```diff
@@ -1,12 +1,14 @@
 package types
 
 import (
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 // WasmKeeper defines the expected interface needed to cam cosmwasm contracts.
 type WasmKeeper interface {
 	HasContractInfo(ctx sdk.Context, contractAddress sdk.AccAddress) bool
+	GetContractInfo(ctx sdk.Context, contractAddress sdk.AccAddress) *wasmtypes.ContractInfo
```

Comment:
unused method

Reply:
It's not.  This method required by task itself (get the contract code id by its address)

Reply:
I can see that you've added this method to two expected wasm keepers: on the interchaintxs module's side and on the contractmanager module's side. the method is used in interchaintxs, but is not in contractmanager
<img width="379" alt="image" src="https://github.com/neutron-org/neutron/assets/34917380/27511e89-e639-43d1-a73a-b6c1160e9c48">